studio/components/AnchorSelect.tsx

```diff
@@ -45,7 +45,9 @@ const AnchorSelect = ({ value, onChange, path }: AnchorSelectProps) => {
     }
 
     try {
-      const response = await client.fetch("*[_id == $id][0]{sections}", {
+      const response = await fetchWithToken<{
+        sections: AnchorItem[];
+      }>("*[_id == $id][0]{sections[]{_key, basicTitle}}", {
```

Comment:
What is the difference between `sections[]` and `sections`?

Reply:
Yes, that part was clear, but the extra `[]`  after `sections` seemed redundant. After double checking is seems to be required, so all good 🤗